### PR TITLE
add deep poverty to poverty charts

### LINKED ISSUE #181: Add deep poverty to poverty chart
as a grouped bar chart like this from the blank slate paper
image

### PR DESCRIPTION
**Pull request by tolaouk**
fixes #181

### CODE REVIEW

**Comment by tolaouk** [comment]
image

image

File: policyengine/impact/population/charts.py [hunk 1]
```diff
@@ -151,29 +151,20 @@ def deep_pov_chg(
...
-    :type reformed: Microsimulation
-    :return: JSON representation of Plotly chart with poverty impact for:
-        - Children (under 18)
-        - Working age adults (18 to State Pension age)
-        - Pensioners (State Pension age and above)
-        - Overall
-    :rtype: dict
-    """
+    metric: str,
+) -> pd.DataFrame:
```
Comment (MaxGhenis) [inline policyengine/impact/population/charts.py:159]: please add a docstring

**Review by MaxGhenis — COMMENTED** [review]
After you get all four colors showing up properly, we'll have to find a way to show them in the legend. I did a quick search and didn't immediately see a way to have multiple colors in the legend square. One option could be [a grouped legend](https://plotly.com/python/legend/#grouped-legend-items), though it's going to take up a lot of room.

Another option could be a radio button to toggle between the poverty rate and deep poverty rate, like the Relative / Absolute decile charts.

**Comment by nikhilwoodruff** [comment]
OK, I'd second the decision to use a radio button: the only argument I can see against it is that we might have wanted to reserve that for a %/million people option like the other rel/abs radio buttons, but I think it's fine to override that.

**Comment by tolaouk** [comment]
image
image

**Comment by MaxGhenis** [comment]
Looks great! What do you think about adding baseline and reform values to the hovercard while you're in here? If it's complicated can also save for a separate PR.

**Comment by tolaouk** [comment]
> Looks great! What do you think about adding baseline and reform values to the hovercard while you're in here? If it's complicated can also save for a separate PR.

I'll try add them here.

**Comment by tolaouk** [comment]
@MaxGhenis something like this?
image
image

**Comment by MaxGhenis** [comment]
Nice! How about this, as to emphasize the value associated with the bar size?
>**Working-age deep poverty falls 6.4%**
from 16.5% to 15.5%

**Comment by tolaouk** [comment]
image
image

**Review by MaxGhenis — APPROVED** [review]

**Comment by MaxGhenis** [comment]
@tolaouk you'll have to update the poverty chart test to add the extra argument. Could you test it for both `is_deep=False` and `is_deep=True`?

**Pull request merged**